.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,19 @@
+name: pull_request
+
+on: [pull_request]
+
+jobs:
+  pull_request:
```

Comment:
The job name is not pull_request nor the file name !

pull_request is just a trigger, better name the file `build-image-test.yaml` and have the job name as `build docker image`.